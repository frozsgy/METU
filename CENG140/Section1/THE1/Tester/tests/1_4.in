7
1 4 4
1 4 5
2 3 3
2 2 3
3 4 2
3 4 1
4 5 3
4 6 3
1 4 3
6 1 1
6 2 1
4 5 6
6 1 2
4 2 2
4 3 3
4 3 1
4 4 2
5 3 2
2 0 5
2 1 6
2 2 4
2 2 5
3 1 5
3 1 4
2 4 4
3 2 3
2 3 4
3 2 4
4 2 3
6 3 4
6 2 4
5 3 3
5 2 3
5 2 2
4 3 2
6 4 2
6 2 2
7 3 0
7 3 1
7 4 2
7 5 2
7 3 4
7 3 2
7 0 0
7 0 1
7 0 2
7 0 3
7 0 4
7 0 6
7 1 0
7 1 1
7 1 2
7 1 3
7 1 4
7 1 5
7 2 0
7 2 1
7 2 2
7 2 3
7 2 4
7 2 5
7 2 6
7 3 0
7 3 1
7 3 3
7 3 4
7 3 5
7 3 6
7 4 0
7 4 1
7 4 2
7 4 3
7 4 4
7 4 5
7 4 6
7 5 0
7 5 1
7 5 2
7 5 3
7 5 4
7 5 5
7 6 0
7 6 1
7 6 2
7 6 3
7 6 4
7 6 5
7 6 6
8 0 0
8 0 1
8 0 2
8 0 3
8 0 4
8 1 0
8 1 1
2 1 2
8 1 4
8 1 5
8 2 0
2 2 1
8 2 6
8 3 0
8 3 1
8 3 4
8 4 0
8 4 1
8 4 2
8 4 4
8 4 5
8 5 0
8 5 1
8 5 3
8 5 4
8 6 0
8 6 1
8 6 3
8 6 4
