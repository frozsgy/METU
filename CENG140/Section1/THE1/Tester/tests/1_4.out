0 0 9 0 0 0 9 
0 0 2 9 0 0 8 
9 2 0 0 9 0 0 
0 9 0 0 0 0 9 
0 9 0 0 9 0 8 
2 8 0 0 9 0 0 
7 2 8 0 0 4 7 
